CMSgov/design-system
# [WNMGDS-1610] Button analytics

## Summary

https://jira.cms.gov/browse/WNMGDS-1610

### Added

- Added analytics tracking for `Button` element. Can be turned on by calling `setButtonSendsAnalytics(true)`.

## How to test

1. `yarn storybook`
2. Navigate to http://localhost:6006/?path=/story/components-button--default-button&globals=analytics:on
3. Click on the button and make sure the event is firing as expected in the console
4. Navigate to http://localhost:6006/?path=/story/components-button--link-button&globals=analytics:on
5. Click on the button and make sure the event is firing as expected in the console
